packager/hls/base/media_playlist.cc

```diff
@@ -22,10 +21,7 @@
 #include "packager/media/base/language_utils.h"
 #include "packager/media/base/muxer_util.h"
 #include "packager/version/version.h"
-
-DEFINE_bool(hls_ext_x_program_date_time,
-            false,
-            "hls print EXT-X-PROGRAM-DATE-TIME tag");
+#include "packager/app/hls_flags.h"
```

Comment:
Move it before line 21, i.e. put it in alphabetical order

Reply:
Done.